wasix-test-run.sh

```diff
@@ -7,7 +7,7 @@ TEST_SERVER=${TEST_SERVER:-localhost:8080}
 ALL_TESTS=( \
   sqlite \
   sqlite-pdo \
-  mail \
+  # mail \
```

Comment:
Why aren't we running the mail tests anymore?

Reply:
Does it work? We don't have any default credentials anymore.

Reply:
We can provide some through env vars. Have an account for testing, and put its credentials here (through GH secrects, probably?)